config/app.php

```diff
@@ -13,7 +13,7 @@
     |
     */
 
-    'name' => env('APP_NAME', 'Laravel'),
+    'name' => env('B-Locker'),
```

Comment:
The `env` function requires the first argument to be the name of the environment: 
```php
env('APP_NAME', 'Laravel')
```
The second argument ('Laravel') is the default value, in case you don't have the `env` variable set.

Reply:
Changed.